WEB-INF/classes/gov/noaa/pfel/erddap/util/EDStatic.java

```diff
@@ -384,6 +384,7 @@ public class EDStatic {
   public static final int DEFAULT_slowDownTroubleMillis = 1000;
   public static final int DEFAULT_unusualActivity = 10000;
   public static final int DEFAULT_updateMaxEvents = 10;
+  public static final int DEFAULT_unusualActivityFailPercent = -1;
```

Comment:
Since this is a new parameter and existing installations will not have it set in their config, this should default to maintain the current behavior (25).